lib/publiccloud/instance.pm

```diff
@@ -278,14 +278,14 @@ Return C<1> true explicit, as stateless and never impact calling code.
 
 sub upload_check_logs_tar {
     my ($self, @files) = @_;
-    my $remote_tar = $autotest::current_test->{name};
-    $remote_tar = "/tmp/" . $remote_tar . "_logs.tar.gz";
-    my $res = $self->ssh_script_output(cmd => 'sudo ls ' . join(' ', @files) . ' 2>/dev/null', proceed_on_failure => 1);
-    return 1 unless ($res);
-
+    my $remote_tar = "/tmp/" . $autotest::current_test->{name} . "_logs.tar.gz";
+    my $res = 'sudo ls -x ' . join(' ', @files) . " 2>/dev/null";
```

Comment:
I believe the `-x` option was the problem here since it generates tabs.  `-1` is a safer option as it output one file per line.

Reply:
Ehm... no, it isn't the -x, because issue started in a version _without_ it 🙂.
I could show many tests each with a different form of **ls** producing always a string with some issue.
After all, I used -x to have an easier horizontal output, without newlines, and split it.